package.json

```diff
@@ -25,9 +25,9 @@
   "author": "Wikimedia service team <services@wikimedia.org>",
   "license": "Apache2",
   "bugs": {
-    "url": "https://github.com/wikimedia/servisor/issues"
+    "url": "https://github.com/wikimedia/service-runner/issues"
```

Comment:
It doesn't exist yet, only now opened https://phabricator.wikimedia.org/T89747 & officially I'm supposed to wait for a few hours before actually creating it.